README.md

```diff
@@ -1,139 +1,169 @@
 # player-loader-webpack-plugin
+The official webpack plugin for the Brightcove Player.
 
 ## Installation
-To install just use
+To install, use:
 
 ```sh
-npm i -D @brightcove/player-loader-webpack-plugin
+npm install --save-dev @brightcove/player-loader-webpack-plugin
 ```
 
-
-## Using the plugin
-
-First require the plugin at the top of your `webpack.config.js`:
+## Basic Usage
+First, require the plugin at the top of your `webpack.config.js`:
 
 ```js
 const PlayerLoader = require('@brightcove/player-loader-webpack-plugin');
 ```
 
-Then create an instance of the `PlayerLoader` plugin in the `plugins` array of your `webpack.config.js`:
+Then create an instance of the `PlayerLoader` plugin in the `plugins` array:
 
-> Note: That `accountId` is a required option!
+> **Note:** `accountId` is a required parameter!
 
 ```js
-  plugins: [
-    new PlayerLoader({accountId: '12345678910'})
-  ],
+plugins: [
+  new PlayerLoader({accountId: '12345678910'})
+]
 ```
 
-For more information on options see the options section of the README.
+For a full list of options, see the [Options section](#options) below.
 
-## Options
+## How it Works
+This webpack plugin will prepend your Brightcove Player to your bundle. Doing this can reduce the number of requests needed by your website and ensure that the global `bc` function is available synchronously.
 
-### `accountId`
-* **REQUIRED**
-* *Type:* `string` | `number`
+### Limitations
+There are several limitations and caveats to using this plugin.
 
-A Video Cloud account ID.
+1. iframe embeds are not supported. This plugin is only suitable for use with in-page/advanced embeds.
+1. If your player is updated or republished, the bundle will need to be re-generated before the bundled player is updated.
 
-### `embedId`
-* *Type:* `string`
-* *Default:* `'default'`
+As an alternative, users who want an iframe embed or want to load their player script asynchronously can use the [@brightcove/player-loader](https://github.com/brightcove/player-loader) project instead, which downloads players at runtime rather than at build time. It does not have these limitations.
```

Comment:
does this mean that this project is instead of the `player-loader`? It would be cool if it's possible to make this project augment player-loader instead of replace it.

Reply:
Well, they're kind of separate use-cases aren't they?

Reply:
No, I always though that you can use player-loader to load the player and then when you want the player script built into your bundle you'd include this webpack plugin. The idea is to reduce friction so if you had to switch from one to the other you have to do extra work.

Reply:
Hmm, good point, that seems like something we could implement pretty easily in the `player-loader` by looking for already-defined players in the bundle rather than downloading them. I'll add a story to our backlog for that.

Reply:
For now, this documentation change is still true, but I think we can easily support coordination between the player-loader and this project. We can update the README again at that point.

Reply:
Yeah, it can just be an enhancement in for this project or player-loader